fineract-provider/build.gradle

```diff
@@ -600,21 +600,3 @@ gitProperties {
 
 // make sure the generateGitProperties task always executes (even when git.properties is not changed)
 generateGitProperties.outputs.upToDateWhen { false }
-
-task prepareConfigJson(dependsOn: 'generateGitProperties') {
```

Comment:
I didn't follow this in full details, but @ptuomola and/or @thesmallstar should review and sign off particularly on this change.. does that conversioning you wanted in the Swagger UI still work after this is removed?

Reply:
I didn't see that this was copied to swagger-ui; I did see that there is a similar file there (though not sure why it's needed there). I could only see that config.json was used as parameter input for the command line generator.

Reply:
Yes - this should be OK - the Swagger UI should only depend on the index.html that has been checked in, not on this config.json file. As far as I recall this is only used for the code generator...